utils/remote-run

```diff
@@ -177,54 +267,85 @@ class ArgumentProcessor(object):
             while self.output_prefix.endswith(posixpath.sep):
                 self.output_prefix = self.output_prefix[:-len(posixpath.sep)]
 
+        if self.input_prefix:
+            split = posixpath.split(self.input_prefix)
```

Comment:
This might require some fixing in the future if we end up supporting remote run on Windows (which would be nice).

Reply:
Agreed. Which might be doable these days given that Windows has an SSH server that (sort of mostly) works.